### ISSUE TITLE
[Windows Pacakaged] Error coming when debuging 

### ISSUE THREAD

**Issue by Darsh0307** [unoplatform/uno#17199]
Windows pacakaged is giving us a wired error when trying to run the application

The application was created using blank project with VSIX wizard. There were no changes made in that 

Uno.Templates.5.3.0-dev.82
VSIX :- 5.3.0.9
"Uno.Sdk": "5.3.0-dev.1495"

```
Copying files: Total 89 mb to layout...
Registering the application to run from layout...
DEP0700: Registration of the app failed. [0x80080204] error 0xC00CE015: App manifest validation error: The app manifest must be valid as per schema: Line 17, Column 140, Reason: The attribute '{http://schemas.microsoft.com/appx/manifest/foundation/windows10}MaxVersionTested' on the element '{http://schemas.microsoft.com/appx/manifest/foundation/windows10}TargetDeviceFamily' is not defined in the DTD/Schema.

========== Build: 1 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========
========== Build completed at 2:46 AM and took 05.019 seconds ==========
========== Deploy: 1 succeeded, 0 failed, 0 skipped ==========
========== Deploy completed at 2:46 AM and took 05.019 seconds ==========

```

<img width="1059" alt="Screenshot 2024-05-25 at 12 26 44 PM" src="https://github.com/unoplatform/Uno.Samples/assets/90713481/f66122d4-c774-4ac8-a468-ba67cfa6bf3d">

<img width="641" alt="Screenshot 2024-05-24 at 8 32 54 PM" src="https://github.com/unoplatform/Uno.Samples/assets/90713481/56e9a4cd-f03e-4ed0-86fb-d91383099deb">
<img width="708" alt="Screenshot 2024-05-25 at 12 18 25 PM" src="https://github.com/unoplatform/Uno.Samples/assets/90713481/808970a5-1666-435d-ac52-99de61fa535d">

**Comment by Darsh0307** [comment]
The issue is still there , The app was created using CLI 
and the templates version is 5.3.4

**Comment by Darsh0307** [comment]
[test.zip](https://github.com/user-attachments/files/15891556/test.zip)

**Comment by jeromelaban** [comment]
@dansiegel generated file:

[Package.appxmanifest.txt](https://github.com/user-attachments/files/15891561/Package.appxmanifest.txt)

**Comment by Darsh0307** [comment]
This one is done.

Solved in 5.3.8 vsix
Tempaltes:- 5.3.5
